gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/util/network/EmergeUploadRequest.kt

```diff
@@ -53,6 +54,7 @@ fun fetchSignedUrl(
   val request: Request = Request.Builder().apply {
     url(url)
     post(body)
+    addHeader("User-Agent", "${SOURCE_GRADLE_PLUGIN}_${BuildConfig.VERSION}")
```

Comment:
@rbro112 snuck this in too

Reply:
Lol you beat my comment https://github.com/EmergeTools/emerge-android/pull/118#discussion_r1499925687